corehq/apps/app_manager/models.py

```diff
@@ -365,7 +366,8 @@ class AutoSelectCase(DocumentSchema):
                         xpath expression.
 
     """
-    mode = StringProperty(choices=[AUTO_SELECT_USER, AUTO_SELECT_FIXTURE, AUTO_SELECT_CASE, AUTO_SELECT_RAW])
+    mode = StringProperty(choices=[AUTO_SELECT_USER, AUTO_SELECT_FIXTURE,
+            AUTO_SELECT_CASE, AUTO_SELECT_LOCATION, AUTO_SELECT_RAW])
```

Comment:
probably don't want to add this here (or we should also update the UI for advanced forms) but for advanced forms you can do the autoload manually via a user data so I don't think we need it.